src/Microsoft.AspNet.Mvc.Razor/Resources.resx

```diff
@@ -121,7 +121,7 @@
     <value>Value cannot be null or empty.</value>
   </data>
   <data name="CompilationFailed" xml:space="preserve">
-    <value>One or more compilation failures occured:</value>
+    <value>One or more compilation failures occurred:</value>
```

Comment:
I think it would be easier to just change that word in English - I bet it's spelled more often incorrectly than correctly :smile: